### PR TITLE
feat(homepage): recent notes as a grid

### PR DESCRIPTION
**Pull request by TatianaFomina**
- Updated Recent notes layout
- Removed redundant typography styles file

image

Resolves https://github.com/codex-team/notes.web/issues/138

### CODE REVIEW

File: src/presentation/pages/Home.vue [hunk 1]
```diff
@@ -1,21 +1,24 @@
 <template>
   <div>
-    <h1>{{ $t('home.title') }}</h1>
+    <Heading :level="1" :class="$style['home__heading']">
+      {{ $t('home.title') }}
+    </Heading>
+
     <div v-if="user">
       <div v-if="noteList">
```
Comment (neSpecc) [inline src/presentation/pages/Home.vue:8]: redundant divs

File: codex-ui/src/vue/components/card/Card.vue [hunk 2]
```diff
@@ -57,7 +59,7 @@ withDefaults(
   align-items: flex-start;
   gap: var(--spacing-l);
```
Comment (neSpecc) [inline codex-ui/src/vue/components/card/Card.vue:60]: gap is 16 in figma

**Review by neSpecc — APPROVED** [review]

File: src/presentation/pages/Home.vue [hunk 3]
```diff
@@ -1,38 +1,28 @@
...
-      <p>{{ $t('home.authText') }}</p>
+    <div v-if="user && !noteList">
+      <p>{{ $t('note.emptyNoteList') }}</p>
+    </div>
+
+    <div v-if="noteList">
+      <div :class="$style['note-list']">
+        <RouterLink v-for="note in noteList.items" :key="note.id" :to="`/note/${note.id}`">
+          <Card :class="$style['note-list__card']" :title="getTitle(note.content)" :subtitle="getSubtitle(note)"
+            orientation="vertical" />
```
Comment (e11sy) [inline src/presentation/pages/Home.vue:19]:
```suggestion
          <Card 
            :class="$style['note-list__card']" 
            :title="getTitle(note.content)" 
            :subtitle="getSubtitle(note)"
            orientation="vertical" />
```

File: src/presentation/pages/Home.vue [hunk 4]
```diff
@@ -1,38 +1,31 @@
...
+    <div v-if="!user">
+      <p>{{ $t('home.authText') }}</p>
     </div>
 
-    <div v-else>
-      <p>{{ $t('home.authText') }}</p>
+    <div v-if="user && noteList?.items.length === 0">
+      <p>{{ $t('note.emptyNoteList') }}</p>
+    </div>
+
```
Comment (e11sy) [inline src/presentation/pages/Home.vue:14]:
```suggestion
    <div v-if="user && noteList?.items.length === 0">
      <p>{{ $t('noteList.emptyNoteList') }}</p>
    </div>

```

File: src/presentation/pages/Home.vue [hunk 5]
```diff
@@ -1,38 +1,31 @@
...
+    <div v-if="noteList">
+      <div :class="$style['note-list']">
+        <RouterLink v-for="note in noteList.items" :key="note.id" :to="`/note/${note.id}`">
+          <Card 
+            :class="$style['note-list__card']"
```
Comment (e11sy) [inline src/presentation/pages/Home.vue:19]:
```suggestion
        <RouterLink
          v-for="note in noteList.items"
          :key="note.id"
          :to="`/note/${note.id}`"
        >
          <Card
            :class="$style['note-list__card']"
```

**Pull request merged**